npm.d.ts

```diff
@@ -0,0 +1,439 @@
+import {LiteralUnion} from '.';
+
+/**
+A person who has been involved in creating or maintaining the package.
+*/
+export type Person =
+	| string
+	| {
+		name: string;
+		url?: string;
+		email?: string;
+	};
+
+export type BugsLocation =
+	| string
+	| {
+		/**
+		The URL to the package's issue tracker.
+		*/
+		url?: string;
+
+		/**
+		The email address to which issues should be reported.
+		*/
+		email?: string;
+	};
+
+export interface DirectoryLocations {
+	/**
+	Location for executable scripts. Sugar to generate entries in the `bin` property by walking the folder.
+	*/
+	bin?: string;
+
+	/**
+	Location for Markdown files.
+	*/
+	doc?: string;
+
+	/**
+	Location for example scripts.
+	*/
+	example?: string;
+
+	/**
+	Location for the bulk of the library.
+	*/
+	lib?: string;
+
+	/**
+	Location for man pages. Sugar to generate a `man` array by walking the folder.
+	*/
+	man?: string;
+
+	/**
+	Location for test files.
+	*/
+	test?: string;
+
+	[directoryType: string]: unknown;
+}
+
+export interface ScriptsDefinition {
```

Comment:
Why the `Definition` postfix?

How about this?

```suggestion
export interface Scripts {
```